search/build.gradle.kts

```diff
@@ -0,0 +1,47 @@
+plugins {
+    alias(libs.plugins.android.library)
+    alias(libs.plugins.kotlin)
+    alias(libs.plugins.hilt)
+    alias(libs.plugins.ksp)
+    alias(libs.plugins.detekt)
+    id("androidx.room")
+}
+
+android {
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+
+    namespace = "com.imashnake.animite.search"
+
+    room {
+        schemaDirectory("$projectDir/schemas")
+    }
+}
+
+kotlin {
+    jvmToolchain(21)
+}
+
+configurations.implementation {
+    exclude(group = "com.intellij", module = "annotations")
```

Comment:
Room introduces duplicate annotation classes as something else, couldn't figure out what else, so just exclude in the model that introduced room :)